Gordon360/Services/NewsService.cs

```diff
@@ -306,6 +327,27 @@ private static bool VerifyUnapproved(StudentNews newsItem)
             return true;
         }
 
+        /// <summary>
+        /// Helper method to verify that a given news item has already been approved
+        /// </summary>
+        /// <param name="newsItem">The news item to verify</param>
+        /// <returns>true ig approved, otherwise throws some kind of meaningful exception</returns>
+        private static bool VerfiyApproved(StudentNews newsItem)
+        {
+            // Note: These checks have been duplicated from StateYourBusiness because we do not want
+            //    SuperAdmins to be able to delete expired news, this should be fixed eventually by
+            //    removing some of the SuperAdmin permissions that are not explicitly given
+            if (newsItem.Accepted == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The news item acceptance status could not be verified." };
```

Comment:
Generally, ResourceNotFound (404) should be reserved for if the item referenced in the API route does not exist (in this case, the student news item itself). Instead, it might be better to use a generic `Exception`. If you change this one, could you also change the one above on line 321?